Decred Wallet/Features/Navigation Menu/NavMenuFloatingButtons.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  NavMenuFloatingButtons.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
```

Comment:
(c) 2019. Please update on the files affected in this PR

Reply:
resolved in 2283e90